src/tsickle.ts

```diff
@@ -779,6 +779,12 @@ class ExternsWriter extends ClosureRewriter {
   /** visit is the main entry point.  It generates externs from a ts.Node. */
   public visit(node: ts.Node, namespace: string[] = []) {
     switch (node.kind) {
+      case ts.SyntaxKind.SourceFile:
```

Comment:
Yes.  We now pass a SourceFile into this visit() function, where before we'd only pass every "declare ..." statement into it.